README.md

```diff
@@ -3,3 +3,8 @@
 # app-operator
 
 The app-operator manages apps in Kubernetes clusters.
+
+## Branches
+
+- `thiccc`
+    - Only to support booting up chart-operator with Helm2 support.
```

Comment:
```suggestion
    - Bootstrap chart-operator in tenant clusters using Helm 2.
```

What about this? 

Can you also add the master and helm3 branches?  We can remove helm3 once we merge it into master.